src/www/containers/contactpage/ContactPage.jsx

```diff
@@ -0,0 +1,128 @@
+import "./ContactPage.scss"
+
+import React, { Component } from "react";
+
+import Input from "../../components/input/Input";
+import axios from "axios";
+import { push } from "connected-react-router";
+import { withTranslation } from "react-i18next";
+
+const top_img = '/img/top_img.png'; //Photo by Patrick Perkins on Unsplash
+const form_img = '/img/form_img.png'; //Photo by LinkedIn Sales Navigator on Unsplash
+
+const clearErrors = () => ({ type: "CLEAR" });
```

Comment:
This function doesn't have any use since it's related to redux. You don't need it anymore!